pcr/gizzmos.py

```diff
@@ -46,4 +46,78 @@ def adjust_volume(self, volume: float) -> str:
         return None, None
 
     def check_connection(self) -> bool:
-        return True
\ No newline at end of file
+        return True
+    
+
+import cv2 as cv
+import numpy as np
+class TestTubeCV:
+    def __init__(self):
+        self.videoCapture = cv.VideoCapture(0, cv.CAP_DSHOW)
+        self.prevCircle = None
+        self.dist = lambda x1, y1, x2, y2: (x1 - x2) ** 2 + (y1 - y2) ** 2
+        self.debug = False
+
+    def predict(self, frame):
```

Comment:
Using python typing, you can say the following:
```
def predict(self, frame) -> List[Tuple[[int, int]]]:
```
This increases code readability so a person looking at this can more easily tell what this function takes in for input and output. You may need to import Tuple and List from typing